en_US/observability/log.md

```diff
@@ -112,7 +112,6 @@ You can also configure EMQX logging through configuration files. For example, if
 log {
   file {
     default {
-      enable = true
```

Comment:
this log handler is disabled by default when starting with forgroud mode, so we need to keep it in the example.